src/compiler/commandLineParser.ts

```diff
@@ -740,6 +740,16 @@ namespace ts {
             description: Diagnostics.Default_catch_clause_variables_as_unknown_instead_of_any,
             defaultValueDescription: false,
         },
+        {
+            name: "inferInstanceTypeArgumentsAsConstraint",

```

Comment:
`strict` flags _usually_ start with `strict`. Given the choice, I think we should do so here.

Reply:
What do you think about "strictDefaultInstanceTypeArguments"?

Reply:
`strictInstanceOfTypeParameters` maybe? Most tightly says what the flag controls.

Reply:
I didn't initially include "instance of" since this change also affects the `#private in obj` check not just `instanceof`, but I agree that it is a good tight description of what it controls. Will switch